add_user.php

```diff
@@ -20,31 +21,32 @@
 			<meta name="Keywords" content="html5, layout, Responsive Design"/>
 			<meta name="Author" content="Faith Tuason"/>
 			<meta name="Description" content="TuneIn Admin - Add New User Page"/>
-
+			
 			<!-- import the webpage's stylesheet goes here -->
 			<link rel="stylesheet" href="css/style.css">
+
+			<!-- import the webpage's javascript -->
+			<!-- nav javascript -->
+			<script src="js/nav.js" defer></script>
 		</head>
+	
         <body>
-             <!-- logo at top of page-->
-			<center><a href="index.php"><img style="width: 100%; max-width: 100px;" src="images/placeholder.png" alt="TuneIn logo"></a></center>	
-			
-			<!-- navigation menu (burger menu)-->
+            <!-- navigation menu-->
 			<?php
 			  require("nav.php");
 			?>
-			
-								<div class="aside"><!-- holds the admin menu with links to the different pages only the admin can access -->
+			<center>
```

Comment:
As I made most of  my html body smaller in width I wanted to position all elements in the center